src/woofmark.js

```diff
@@ -218,7 +218,21 @@ function woofmark (textarea, options) {
       if (currentMode === 'html') {
         textarea.value = parse('parseHTML', textarea.value).trim();
       } else {
-        textarea.value = parse('parseHTML', editable).trim();
+        var regexp = /\*\*[A-Z][^*]+ \*\*/gi;
```

Comment:
Can we have a better name for this `regex`?  Instead of using regex and regex2 as variable name, can we have a separate `utils` file where each regex is defined (can be done in a separate PR)

Reply:
was intending to comment the code, and will think of something more suitable to rename the variable with like your're suggesting.  About a utils file, do you mean putting this code in a function and importing from a utils file?

Reply:
Yes only this regex line but it can be done later. No worries 👍